app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -153,6 +167,39 @@ def gather_data_for_this_region(arm_service, method = "list")
         results.flatten
       end
 
+      def parse_security_group(security_group)
+        uid = security_group.id
+
+        description = [
+          security_group.name,
+          security_group.resource_group,
+          security_group.location
+        ].join(' - ')
+
+        new_result = {
+          :type           => self.class.security_group_type,
+          :ems_ref        => uid,
+          :name           => security_group.name,
+          :description    => description,
+          :firewall_rules => parse_firewall_rules(security_group)
+        }
+
+        return uid, new_result
+      end
+
+      def parse_firewall_rules(sg)
+        sg.properties.security_rules.map do |rule|
+          {
+            :name            => rule.name,
+            :host_protocol   => rule.properties.protocol.upcase,
+            :port            => rule.properties.destination_port_range.split('-').first,
+            :end_port        => rule.properties.destination_port_range.split('-').last,
+            :direction       => rule.properties.direction,
+            :source_ip_range => rule.properties.source_address_prefix
+          }
```

Comment:
@bronaghs I assume that's just the Rails security group id? The "source" prefix was a bit confusing to me.